### DISCUSSION TITLE
Configs

### DISCUSSION THREAD

**Discussion by cartiqo** [LunarTaku/djs-ticket-system#6] [Ideas]
I think you should make something where you can add support roles and add up to 5 buttons and just make a custom description if that's possible father

**Comment by LunarTaku** [comment]
good idea! Im gonna code it now. But when you said "make a custom description" did you mean the panel description? Or the created ticket description.

**Reply by cartiqo** [reply]
Yeah like a panel description and maybe also a claim ticket button and after it has been used it could be disabled

**Reply by LunarTaku** [reply]
ight
